# Michael Alonge
Software Engineer & Computational Biologist

**Email:** malonge11@gmail.com  
**Phone:** +1 (831) 201-2788  
**Location:** Los Angeles, CA  
**Website:** https://malonge.com  
**LinkedIn:** https://www.linkedin.com/in/michael-alonge-1348a973/  
**GitHub:** https://github.com/malonge  
**Google Scholar:** https://scholar.google.com/citations?user=5zUWawIAAAAJ&hl=en  

## Professional Summary

Experienced professional with expertise in [your field]. Passionate about [key interests] and committed to delivering [value proposition].

## Professional Experience

### Lead Software Engineer, Computational Biology
**Ohalo Genetics** | Mar 2025 - Present (Present)

I lead the design and development of the genomics data platform for Ohalo Genetics responsible for automated and scalable management and analysis of genomics data supporting breeding, gene editing, and scientific IP. I also serve as the lead subject matter expert for genomics and bioinformatics with an emphasis on scalable and accurate genotyping methods.

**Key Achievements:**
• Lead the development of the genomics data platform for Ohalo Genetics
• Manage and mentor a team of software engineers and bioinformaticians
• Lead the strategic direction of genomics supporting gene editing, breeding, and IP

### Lead Computational Biologist
**Ohalo Genetics** | Dec 2021 - Mar 2025 (3y 3m)

Brief description of role and company

**Key Achievements:**
• Lead bioinformatics and genomics analysis for Ohalo's Boosted Breeding™ foundational patents.
• Developed Ohalo's first scalable genotyping pipeline supporting breeding and gene editing.
• Lead bioinformatics and genomics analysis for all programs, like Ohalo's 'FruitionOne' Almond program.

### Ph.D. Student, Computer Science
**Johns Hopkins University** | Aug 2017 - May 2021 (3y 8m)

I completed my Ph.D. in Computer Science at Johns Hopkins University under the supervision of Professor Michael Schatz. My research focused on advanced applications of genome assembly and structural genetic variation in both humans and plants. I served as a teaching assistant for 'Computational Genomics: Sequences', taught by Professor Ben Langmead.

**Key Achievements:**
• Published several first author papers in journals such as Cell and Science
• Inited speaker at a Galaxy webinar and the Boyce Thompson Institute for Plant Research

### Research Associate I and II: Computational Biology
**Driscoll's** | Oct 2014 - Jul 2017 (2y 9m)

I led computational biology projects with the ultimate goal of assisting breeders and molecular biologists. I especially focused on establishing reference genome assemblies to facilitate downstream genomics analyses. I also assisted with wet-lab protocols and plant phenotyping.

**Key Achievements:**
• Led computational biology projects with the ultimate goal of assisting breeders and molecular biologists
• Established reference genome assemblies to facilitate downstream genomics analyses
• Assisted with wet-lab protocols and plant phenotyping

## Education

### Ph.D. in Computer Science
**Johns Hopkins University** | Aug 2017 - May 2021

### Bachelor of Science in Biomolecular Engineering
**University of California, Santa Cruz** | Aug 2009 - May 2014

## Publications

### Solanum pan-genetics reveals paralogues as contingencies in crop engineering
**Nature Publishing Group UK London** | 2025

Published in Nature. Co-authored with Benoit, Matthias; Jenike, Katharine M; Satterlee, James W; Ramakrishnan, Srividya; Gentile, Iacopo; Hendelman, Anat; Passalacqua, Michael J; Suresh, Hamsini; Shohat, Hagai; Robitaille, Gina M.
**Publication:** https://doi.org/10.1038/s41586-025-08619-6

### The complete sequence of a human genome
**American Association for the Advancement of Science** | 2022

Published in Science, Volume 376, Issue 6588, Pages 44-53. Co-authored with Nurk, Sergey; Koren, Sergey; Rhie, Arang; Rautiainen, Mikko; Bzikadze, Andrey V; Mikheenko, Alla; Vollger, Mitchell R; Altemose, Nicolas; Uralsky, Lev; Gershman, Ariel.
**Publication:** https://doi.org/10.1126/science.abj6987

### Chasing perfection: validation and polishing strategies for telomere-to-telomere genome assemblies
**Nature Publishing Group** | 2022

Published in Nature Methods, Pages 687-695. Co-authored with Mc Cartney, Ann M; Shafin, Kishwar; Bzikadze, Andrey V; Formenti, Giulio; Fungtammasan, Arkarachai; Howe, Kerstin; Jain, Chirag; Koren, Sergey; Logsdon, Glennis A.
**Publication:** https://doi.org/10.1038/s41592-022-01440-3

### Establishing Physalis as a new Solanaceae model system enables genetic reevaluation of the inflated calyx syndrome
**Oxford University Press** | 2023

Published in The Plant Cell, Pages 351-368. Co-authored with He, Jia; Ramakrishnan, Srividya; Benoit, Matthias; Soyk, Sebastian; Reem, Nathan T; Hendelman, Anat; Van Eck, Joyce; Schatz, Michael C; Lippman, Zachary B.
**Publication:** https://doi.org/10.1093/plcell/koac305

### Automated assembly scaffolding using RagTag elevates a new tomato system for high-throughput genome editing
**BioMed Central** | 2022

Published in Genome Biology, Volume 23, Issue 1, Page 258. Co-authored with Lebeigle, Ludivine; Kirsche, Melanie; Jenike, Katie; Ou, Shujun; Aganezov, Sergey; Wang, Xingang; Lippman, Zachary B; Schatz, Michael C; Soyk, Sebastian.
**Publication:** https://doi.org/10.1186/s13059-022-02823-7

### Natural genetic diversity in tomato flavor genes
**Frontiers** | 2021

Published in Frontiers in Plant Science, Volume 12, Page 642828. Co-authored with Pereira, Lara; Sapkota, Manoj; Zheng, Yi; Zhang, Youjun; Razifard, Hamid; Taitano, Nathan K; Schatz, Michael; Fernie, Alisdair; Wang, Ying.
**Publication:** https://doi.org/10.3389/fpls.2021.642828

### The genetic and epigenetic landscape of the Arabidopsis centromeres
**American Association for the Advancement of Science** | 2021

Published in Science, Volume 374, Issue 6569, Page eabi7489. Co-authored with Naish, Matthew; Wlodzimierz, Piotr; Tock, Andrew J; Abramson, Bradley W; Schmücker, Anna; Mandáková, Terezie; Jamge, Bhagyshree; Lambing, Christophe; Kuo, Pallas.
**Publication:** https://doi.org/10.1126/science.abi7489

### Pan-genomics and the structural diversity of plant genomes
**Johns Hopkins University** | 2021

Ph.D. Dissertation. Published by Johns Hopkins University.
**Publication:** http://jhir.library.jhu.edu/handle/1774.2/66801

### Major Impacts of Widespread Structural Variation on Gene Expression and Crop Improvement in Tomato
**Elsevier** | 2020

Published in Cell, Volume 182, Issue 1, Pages 145-161.e23. Co-authored with Wang, Xingang; Benoit, Matthias; Soyk, Sebastian; Pereira, Lara; Zhang, Lei; Suresh, Hamsini; Ramakrishnan, Srividya; Maumus, Florian; Ciren, Danielle.
**Publication:** https://doi.org/10.1016/j.cell.2020.05.021

### Chromosome-Scale Assembly of the Bread Wheat Genome Reveals Thousands of Additional Gene Copies
**Genetics Soc America** | 2020

Published in Genetics, Volume 216, Issue 2, Pages 599-608. Co-authored with Shumate, Alaina; Puiu, Daniela; Zimin, Aleksey; Salzberg, Steven L.
**Publication:** https://doi.org/10.1534/genetics.120.303501

### A master regulator of regeneration
**American Association for the Advancement of Science** | 2019

Published in Science, Volume 363, Issue 6432, Pages 1152-1153. Co-authored with Schatz, Michael C.
**Publication:** https://doi.org/10.1126/science.aaw6258

### Duplication of a domestication locus neutralized a cryptic variant that caused a breeding barrier in tomato
**Nature Publishing Group** | 2019

Published in Nature Plants, Volume 5, Issue 5, Pages 471-479. Co-authored with Soyk, Sebastian; Lemmon, Zachary H; Sedlazeck, Fritz J; Jiménez-Gómez, José M; Hutton, Samuel F; Van Eck, Joyce; Schatz, Michael C; Lippman, Zachary B.
**Publication:** https://doi.org/10.1038/s41477-019-0422-z

### Accurate circular consensus long-read sequencing improves variant detection and assembly of a human genome
**Nature Publishing Group** | 2019

Published in Nature Biotechnology, Volume 37, Issue 10, Pages 1155-1162. Co-authored with Wenger, Aaron M; Peluso, Paul; Rowell, William J; Chang, Pi-Chuan; Hall, Richard J; Concepcion, Gregory T; Ebler, Jana; Fungtammasan, Arkarachai; Kolesnikov, Alexey; Olson, Nathan D.
**Publication:** https://doi.org/10.1038/s41587-019-0217-9

### The bracteatus pineapple genome and domestication of clonally propagated crops
**Nature Publishing Group** | 2019

Published in Nature Genetics, Volume 51, Issue 10, Pages 1549-1558. Co-authored with Chen, Li-Yu; VanBuren, Robert; Paris, Margot; Zhou, Hongye; Zhang, Xingtan; Wai, Ching Man; Yan, Hansong; Chen, Shuai; Ramakrishnan, Srividya.
**Publication:** https://doi.org/10.1038/s41588-019-0506-8

### RaGOO: fast and accurate reference-guided scaffolding of draft genomes
**BioMed Central** | 2019

Published in Genome Biology, Volume 20, Issue 1, Page 224. Co-authored with Soyk, Sebastian; Ramakrishnan, Srividya; Wang, Xingang; Goodwin, Sara; Sedlazeck, Fritz J; Lippman, Zachary B; Schatz, Michael C.
**Publication:** https://doi.org/10.1186/s13059-019-1829-6

### The genome sequence and transcriptome of Potentilla micrantha and their comparison to Fragaria vesca (the woodland strawberry)
**Oxford University Press** | 2018

Published in GigaScience, Volume 7, Issue 4, Pages 1-14. Co-authored with Buti, Matteo; Moretto, Marco; Barghini, Elena; Mascagni, Flavia; Natali, Lucia; Brilli, Matteo; Lomsadze, Alexandre; Sonego, Paolo; Giongo, Lara.
**Publication:** https://doi.org/10.1093/gigascience/giy010

## Skills

**Technical Skills** (Advanced)
Skill 1, Skill 2, Skill 3, Skill 4, Skill 5

**Programming Languages** (Advanced)
Python, Language 2

**Tools & Technologies** (Intermediate)
Tool 1, Tool 2, Tool 3

## Projects

### Project Name
Dec 2022 - May 2023

Brief description of the project

**Key Highlights:**
• Project achievement 1
• Project achievement 2

**Project URL:** https://project-url.com

## Languages

• **English:** Native speaker
• **Spanish:** Fluent

## Interests

### Professional Interests
Interest 1, Interest 2, Interest 3

## References

### Reference Name
Reference description and contact information.

